gee-community/qgis-earthengine-plugin
# Feature/ci

## What I Changed

- Add CI for unit tests
- Rename sentinel file to example
- Add pre-commit and integrate to CI
- Implement fixes to ruff suggestions
- Add ignore comments for QA from ruff
- Update relative imports

## How to test it

- Run examples within local version of QGIS
- See github action execution

## Other notes

- Closes #177 
- Closes #176 
- Closes #175
